Sources/SwiftInspectorVisitors/FunctionDeclarationVisitor.swift

```diff
@@ -42,6 +48,7 @@ fileprivate final class FunctionSignatureVisitor: SyntaxVisitor {
 }
 
 public struct FunctionDeclarationInfo: Codable, Hashable {
+  public let modifiers: [String]
```

Comment:
Our `PropertyInfo` has a strongly typed `Modifiers` that isn't directly applicable here (since properties have modifiers which don't apply to functions like `private(set)`). We could create a new strongly typed `Modifiers` for use in function declarations, or we could just stick to a `String` array for now. I think I prefer the simplicity of working with Strings at this level, but maybe I'm open to being convinced that I should use a stronger type.

Reply:
I'd prefer to use an array of `Modifier` similar to the PropertyInfo (we likely wanna move that to not be nested inside `PropertyInfo `). I don't see a modifier not being available to a function as problematic since this is not enums, but rather an `OptionSet` and not everything in an OptionSet IMO needs to be available to everything.

The nice thing about the Modifier is that we can have multiple things all at once like `static` and `public` in the same type, and if we want to move to a String from there its trivial, but doing it the other way around (from string -> single modifiers) requires us to parse the String again, which seems error prone.

Reply:
This is now a breaking change, and I've unified our usage of `Modifiers`. Let me know what you think!